docs/azure_arc_jumpstart/azure_arc_vsphere/resource_bridge/powershell/_index.md

```diff
@@ -0,0 +1,197 @@
+---
+type: docs
+title: "Connect VMware vCenter Server to Azure Arc using PowerShell"
+linkTitle: "Connect VMware vCenter Server to Azure Arc using PowerShell"
+weight: 1
+description: >
+---
+
+## Connect VMware vCenter Server to Azure Arc using PowerShell
+
+The following README will guide you on how to use the provided PowerShell script to deploy the [Azure Arc resource bridge](https://docs.microsoft.com/en-us/azure/azure-arc/resource-bridge/overview) in your vSphere environment to connect your vCenter Server to Azure Arc.
+
+> **NOTE:  This guide will not provide instructions on how to deploy and set up your VMware environment, it must be already provisioned.**
+
+## Prerequisites
+
+- Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+- Create Azure service principal (SP)
+
+    To be able to complete the scenario and its related automation, an Azure service principal assigned with the “Contributor” role is required. To create it, login to your Azure account using PowerShell and run the below command.
+
+    ```powershell
+    Connect-AzAccount
+    $sp = New-AzADServicePrincipal -DisplayName "<Unique SP Name>" -Role 'Contributor'
+    ```
+
+    For example:
+
+    ```powershell
+    $sp = New-AzADServicePrincipal -DisplayName "AzureArcvSphere" -Role 'Contributor'
+    ```
+
+    This command will create a variable with a secure string as shown below:
+
+    ```shell
+    Secret                : System.Security.SecureString
+    ServicePrincipalNames : {XXXXXXXXXXXXXXXXXXXXXXXXXXXX, http://AzureArcvSphere}
+    ApplicationId         : XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    ObjectType            : ServicePrincipal
+    DisplayName           : AzureArcvSphere
+    Id                    : XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    Type                  :
+    ```
+
+    To expose the generated password use the below code to export the secret:
+
+    ```powershell
+    $BSTR = [System.Runtime.InteropServices.Marshal]::SecureStringToBSTR($sp.Secret)
+    $UnsecureSecret = [System.Runtime.InteropServices.Marshal]::PtrToStringAuto($BSTR)
+    ```
+
+    Copy and save the Service Principal ApplicationId and Secret as you will need it for later on in the automation.
+
+    > **NOTE: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/powershell/module/az.resources/new-azadserviceprincipal?view=azps-5.4.0)**
+
+- Get the ConnectedVMwarevSphere resource provider Id.
+
+  ```shell
+  az ad sp show --id 'ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7' --query '{name:displayName,objectId:objectId}'
+  ```
+  
+    ![Screenshot of Azure resource provider for vSphere](./01.png)
+
+### vCenter Prerequisites
+
+- A vCenter Server with version 6.7.
```

Comment:
```suggestion
- vCenter Server v6.7.
```

Reply:
Either here or in the preview note suggested, I would add a comment around vSphere 7.x support